Assets/Scripts/Platform.cs

```diff
@@ -21,9 +20,12 @@ public class Platform : MonoBehaviour
     public void CollectPickUp(PickUp pickUp) {
         outstandingPickUps.Remove(pickUp);
         pickUpsCollected++;
-        General.Instance.OnPickUpCollected();
+        if (General.Instance != null) {
```

Comment:
Again, I don't like this check. We shouldn't be allowed to get into this situation, right?